generated/nifgen/enums.py

```diff
@@ -4,48 +4,48 @@
 
 
 class AnalogPath(Enum):
```

Comment:
I'm pretty sure this enum is external calibration-specific.

Reply:
Do we need to support removing enums? Or is not referencing them anywhere enough?

Reply:
Need? That's up for debate.
Should? It's dead code and it shows up in documentation. I think so.

Reply:
Issue #573